src/main/java/net/dv8tion/jda/api/interactions/components/selections/SelectOption.java

```diff
@@ -30,6 +30,21 @@
  */
 public class SelectOption implements SerializableData
 {
+    /**
+     * The maximum length a select option label can have
+     */
+    public static final int LABEL_MAX_LENGTH = 25;
```

Comment:
Add `@see` tag to all constants

Reply:
SelectOption has neither a builder nor a setter. Do you want me to reference the copy constructors withLabel, withValue and withDiscription?